app/helpers/favourites_helper.rb

```diff
@@ -1,9 +1,13 @@
 module FavouritesHelper
```

Comment:
It is better to use `link_to` here instead of the `button_to`, you can style link to look lite a button with bootstrap calsses